tests/manual-test-cases/Group5-Functional-Tests/5-30-Docker-Compose-ELK.robot

```diff
@@ -0,0 +1,108 @@
+# Copyright 2016-2019 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-30 - docker compose ELK
+Resource  ../../resources/Util.robot
+Suite Setup  Deploy Environment
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Variables ***
+${NIMBUS_LOCATION}  sc
+${NIMBUS_LOCATION_FULL}  NIMBUS_LOCATION=${NIMBUS_LOCATION}
+
+*** Keywords ***
+Deploy Environment
```

Comment:
I wonder if we could extract this step into a common step so other tests may also use this type of cluster with iscsi?

Reply:
Currently not needed.